packages/react-components/src/components/VideoGallery/LocalVideoCameraButton.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IconButton } from '@fluentui/react';
+import React from 'react';
+import { OptionsDevice } from '../DevicesButton';
+
+/**
+ * @internal
+ */
+export interface LocalVideoCameraCycleButtonProps {
+  /** Array of cameras available to the user. */
+  cameras: OptionsDevice[];
+  /** Currently selected camera in the local video stream. */
+  selectedCamera: OptionsDevice;
+  /** callback function to change video feed. */
+  onSelectCamera: (device: OptionsDevice) => Promise<void>;
+}
+
+const localVideoCameraCycleButtonStyles: IButtonStyles = {
+  root: {
+    position: 'absolute',
+    right: '0.2rem',
+    top: '0.2rem',
+    color: '#FFFFFF', // only shows up on running video feed to we want to force specific colours.
+    zIndex: 5, // to layer the button above the video tile.
+    background: 'transparent'
+  },
+  rootFocused: {
```

Comment:
for a11y reasons there will need to be some indication this component has focus - is there a border or anything that happens on focus?

Reply:
do we have a design suggestion for this? (I'll ask Alex P but do we have a standard we apply now?) right now it was just making the background completely white over the video